_sass/search.scss

```diff
@@ -316,6 +316,11 @@ input::placeholder {
   }
 }
 
+.main-menu #search-icon {
+  background-size: 14px 16px;
+  background-position: 6px 5px;
+}
+
```

Comment:
I don't think you need to wrap these changes in `main-menu`. You can edit the values in the `search-icon` block.